woocommerce/woocommerce
# Add aria-label for simple select dropdown in select-control component

### All Submissions:


### Changes proposed in this Pull Request:

Adds `aria-label` to improve accessibility for simple select dropdown. 

Closes https://github.com/woocommerce/team-ghidorah/issues/136 

### How to test the changes in this Pull Request:

1. Start OBW.
2. Select a country and inspect the country list element. 
3. Confirm the input has `aria-label`





### Other information:




### FOR PR REVIEWER ONLY:
